test/acceptance/step_definitions/payment/payment.steps.js

```diff
@@ -0,0 +1,39 @@
+const { Given, When, Then, Before, setDefaultTimeout } = require('@cucumber/cucumber')
+const { sendMessage, messageReciever, clearSubscription } = require('../../support/message-service')
+const config = require('../../support/config')
+const request = require('../../fixtures/payment-request')
+const __ = require('hamjest')
+
+setDefaultTimeout(60 * 1000)
+
+Before({ name: 'Clear topic to ensure clean test run' }, async function () {
```

Comment:
Suggest changing the style of the function declaration to match the rest of the code. ie

async () => instead of async function ()